keywords/LiteServJavaAndroid.py

```diff
@@ -0,0 +1,133 @@
+        else:
+            url = "{}/couchbase-lite-android/{}/{}/{}".format(
+                LATEST_BUILDS, version, self.version_build, package_name)
+
+        log_info("Downloading {} -> {}/{}".format(url, BINARY_DIR, package_name))
+        resp = requests.get(url)
+        resp.raise_for_status()
+        with open("{}/{}".format(BINARY_DIR, package_name), "wb") as f:
+            f.write(resp.content)
+
+    def install(self):
+        if self.storage_engine == "SQLite":
+            apk_name = "couchbase-lite-android-liteserv-SQLite-{}-debug.apk".format(
+                self.version_build)
+        else:
+            apk_name = "couchbase-lite-android-liteserv-SQLCipher-\
+            ForestDB-Encryption-{}-debug.apk".format(self.version_build)
+
+        apk_path = "{}/{}".format(BINARY_DIR, apk_name)
+        self.install_apk(apk_path, "com.couchbase.liteservandroid")
+
+    def stop(self):
+        self.stop_activity("com.couchbase.liteservandroid")
+
+    def remove(self):
+        self.remove_apk("com.couchbase.liteservandroid")
+
+    def launch_and_verify(self):
+
+        encryption_enabled = False
+        if self.storage_engine == "SQLCipher" or self.storage_engine == "ForestDB+Encryption":
+            encryption_enabled = True
+
+        if encryption_enabled:
+            log_info("Encryption enabled ...")
+
+            # Build list of dbs used in the tests and pass them to the activity
+            # to make sure the dbs are encrypted during the tests
+            db_flags = []
+            for db_name in REGISTERED_CLIENT_DBS:
+                db_flags.append("{}:pass".format(db_name))
+            db_flags = ",".join(db_flags)
+
+            log_info("Running with db_flags: {}".format(db_flags))
+
+            if self.storage_engine == "SQLCipher":
+                output = subprocess.check_output([
+                    "adb", "shell", "am", "start", "-n", self.activity_name,
+                    "--es", "username", "none",
+                    "--es", "password", "none",
+                    "--ei", "listen_port", str(self.port),
+                    "--es", "storage", "SQLite",
+                    "--es", "dbpassword", db_flags
+                ])
+                log_info(output)
+            elif self.storage_engine == "ForestDB+Encryption":
+                output = subprocess.check_output([
+                    "adb", "shell", "am", "start", "-n", self.activity_name,
+                    "--es", "username", "none",
+                    "--es", "password", "none",
+                    "--ei", "listen_port", str(self.port),
+                    "--es", "storage", "ForestDB",
+                    "--es", "dbpassword", db_flags
+                ])
+                log_info(output)
+        else:
+            log_info("No encryption ...")
+            output = subprocess.check_output([
+                "adb", "shell", "am", "start", "-n", self.activity_name,
```

Comment:
Can we use launch_args and launch_args.extend as in https://github.com/couchbaselabs/mobile-testkit/pull/988/files